web/views.py

```diff
@@ -222,6 +224,9 @@ class StudiesHistoryView(LoginRequiredMixin, generic.ListView):
     template_name = "web/studies-history.html"
     model = Study
 
+    # i18n for redirect
+    login_url = '/' + get_language() + '/login.html'
+
```

Comment:
This might be unnecessary.  The build in translation view appears to be working.  For example, https://lookit-staging.mit.edu/es/login/.  Maybe I am misunderstanding something.